### PR TITLE
Update download instructions and sumarize them in a central place

### PR DESCRIPTION
**Pull request by cmnrd**
This PR updates the download page (e.g. it includer instructions for installing via AUR), and it reduces redundancy by bundling all download instructions on the download page. Consequently, the scope of the tools sections in the handbook changes, and they should now only provide usage instructions. With this change, they look really empty... and we should make an effort to fill those pages in subsequent PR.

I also removed the Troubleshooting section for Python, as I believe this is outdated and most of it, should be fixed by generating launch scripts in bin.

### CODE REVIEW

**Comment by lhstrh** [comment]
I think this makes sense. We should go into more detail about using the tools on those other pages; I agree that just padding those with redundant installation instructions isn't very helpful...

**Review by edwardalee — CHANGES_REQUESTED** [review]
The command-line installation tools do not include how to install the nightly build. They should.  For the VS Code instructions, the instructions for switching to the pre-release version should be in their own paragraph.

**Comment by cmnrd** [comment]
Yes, I hadn't updated the CLI instructions yet. The PR is now ready for review (please see the updated PR description). I also updated the VS Code section as requested by Edward and made some additional changes to math the style with the other sections.

File: packages/lingua-franca/src/templates/pages/download.tsx [hunk 1]
```diff
@@ -42,40 +42,92 @@ const Index: React.FC<Props> = (props) => {
...
     <div className="raised main-content-block">
     <h2 id="vscode">Visual Studio Code</h2>
-      <p>Our Visual Studio Code extension is available on the <
-        a href="https://marketplace.visualstudio.com/items?itemName=lf-lang.vscode-lingua-franca">Visual Studio Marketplace</a
-        > and <a href="https://open-vsx.org/extension/lf-lang/vscode-lingua-franca">VSX Registry</a>.
-        To install this extension from the marketplace, launch VS Code Quick Open (<kbd>Ctrl</kbd> + <kbd>P</kbd>) and enter:<p><code>ext install lf-lang.vscode-lingua-franca</code></p>
-      </p><p>  
-        You can also run:
-        <p><code>code --install-extension lf-lang.vscode-lingua-franca</code></p> in your terminal to install the extension. To use the nightly pre-release of the extension instead of the latest release, find the Lingua Franca extension in the Extensions tab and click on the "Switch to Pre-Release Version" button.
+      <p>Our Visual Studio Code extensios can be installed via the Marketplace or build from source as detailed below.
```
Comment (lhstrh) [inline packages/lingua-franca/src/templates/pages/download.tsx:45]:
```suggestion
      <p>Our Visual Studio Code extension can be installed via the Marketplace or build from source as detailed below.
```

File: packages/lingua-franca/src/templates/pages/download.tsx [hunk 2]
```diff
@@ -42,40 +42,92 @@ const Index: React.FC<Props> = (props) => {
...
+        <p>The Lingua Franca extension is available on the <a href="https://marketplace.visualstudio.com/items?itemName=lf-lang.vscode-lingua-franca">Visual Studio Marketplace</a> and the <a href="https://open-vsx.org/extension/lf-lang/vscode-lingua-franca">VSX Registry</a>. To install the extension, open VS Code, launch Quick Open (<kbd>Ctrl</kbd> + <kbd>P</kbd>) and enter:
+          <p><code>ext install lf-lang.vscode-lingua-franca</code></p>
+        </p><p>
+          Alternatively, you can run the following command in your terminal:
+          <p><code>code --install-extension lf-lang.vscode-lingua-franca</code></p>
+        </p><p>To use the nightly pre-release of the extension instead of the latest release, find the Lingua Franca extension in the Extensions tab and click on the "Switch to Pre-Release Version" button.</p>
+      </div>
+      <div style={{borderTop: "1px lightgray solid"}}>
+          <h3>From Source</h3>
+          <p>Please refer to the <a href="https://github.com/lf-lang/vscode-lingua-franca">Lingua Franca Code Extension GitHub repository</a> for build instructions.</p>
```
Comment (lhstrh) [inline packages/lingua-franca/src/templates/pages/download.tsx:59]:
```suggestion
          <p>Please refer to the <a href="https://github.com/lf-lang/vscode-lingua-franca">Lingua Franca VS Code GitHub repository</a> for build instructions.</p>
```

File: packages/lingua-franca/src/templates/pages/download.tsx [hunk 3]
```diff
@@ -42,40 +42,92 @@ const Index: React.FC<Props> = (props) => {
...
+              <p><code>curl -Ls https://install.lf-lang.org | sh -s cli</code></p>
+          </p>
+          <p>You can also install the nightly pre-release:
+              <p><code>curl -Ls https://install.lf-lang.org | sh -s cli nightly</code></p>
+          </p>
+          <p>You can use the <code>--prefix=&lt;path&gt;</code> argument to change the default install location.</p>
+      </div>
+      <div style={{borderTop: "1px lightgray solid"}}>
+          <h3>AUR</h3>
+          <p>There are binary packages availabe in the Arch user repository, which you can install using your favourite AUR helper. For instance, with yay:
```
Comment (lhstrh) [inline packages/lingua-franca/src/templates/pages/download.tsx:114]:
```suggestion
          <p>There are binary packages available in the Arch user repository, which you can install using your favorite AUR helper. For instance, with <code>yay</code>:
```

**Review by lhstrh — COMMENTED** [review]
Looks good like a good reorganization but I noticed a bunch of typos just by reading through it. I would run this through a spell checker before merging it.

**Comment by cmnrd** [comment]
Hmm, looks like my normal spell checker doesn't like .tsx files. I run the changes through an external checker and pushed several fixes.

**Comment by lhstrh** [comment]
@edwardalee: can we go ahead with merging this, or is there still anything you'd like to see changed?

**Review by edwardalee — APPROVED** [review]
OK, Looks good to me.

**Pull request merged**